Assets/PatchKit Patcher/Scripts/AppStarter.cs

```diff
@@ -26,12 +27,30 @@ public AppStarter(App app)
             AppFinder = new AppFinder();
         }
 
+        private string ResolveExecutablePath(AppVersion appVersion)
+        {
+            if (!string.IsNullOrEmpty(appVersion.MainExecutable))
+            {
+                return Path.Combine(_app.LocalDirectory.Path, appVersion.MainExecutable);
```

Comment:
Maybe check if that file exist before returning it as valid executable path?

That would be much more safe.

Reply:
I agree. Then report this incident to sentry and fallback to executable search.